packages/app-store/video.adapters.generated.ts

```diff
@@ -3,18 +3,19 @@
     Don't modify this file manually.
 **/
 export const VideoApiAdapterMap =
-  process.env.NEXT_PUBLIC_IS_E2E === "1"
-    ? {}
-    : {
-        dailyvideo: import("./dailyvideo/lib/VideoApiAdapter"),
-        huddle01video: import("./huddle01video/lib/VideoApiAdapter"),
-        jelly: import("./jelly/lib/VideoApiAdapter"),
-        jitsivideo: import("./jitsivideo/lib/VideoApiAdapter"),
-        nextcloudtalk: import("./nextcloudtalk/lib/VideoApiAdapter"),
-        office365video: import("./office365video/lib/VideoApiAdapter"),
-        shimmervideo: import("./shimmervideo/lib/VideoApiAdapter"),
-        sylapsvideo: import("./sylapsvideo/lib/VideoApiAdapter"),
-        tandemvideo: import("./tandemvideo/lib/VideoApiAdapter"),
-        webex: import("./webex/lib/VideoApiAdapter"),
-        zoomvideo: import("./zoomvideo/lib/VideoApiAdapter"),
-      };
+  // process.env.NEXT_PUBLIC_IS_E2E === "1"
+  //   ? {}
+  //   :
+  {
+    dailyvideo: import("./dailyvideo/lib/VideoApiAdapter"),
+    huddle01video: import("./huddle01video/lib/VideoApiAdapter"),
+    jelly: import("./jelly/lib/VideoApiAdapter"),
+    jitsivideo: import("./jitsivideo/lib/VideoApiAdapter"),
+    nextcloudtalk: import("./nextcloudtalk/lib/VideoApiAdapter"),
+    office365video: import("./office365video/lib/VideoApiAdapter"),
+    shimmervideo: import("./shimmervideo/lib/VideoApiAdapter"),
+    sylapsvideo: import("./sylapsvideo/lib/VideoApiAdapter"),
+    tandemvideo: import("./tandemvideo/lib/VideoApiAdapter"),
+    webex: import("./webex/lib/VideoApiAdapter"),
+    zoomvideo: import("./zoomvideo/lib/VideoApiAdapter"),
+  };
```

Comment:
Do we need to change it? @Udit-takkar

Reply:
@Udit-takkar this seems unintentional, can you confirm?